### PR TITLE
fix: set proper borders / alignment in quarkus wizard

### LINKED ISSUE #636: Project settings wizard page has no (left nor right) borders
**Steps:**
1. EXEC: launch IDEA, hit "New Project", select "Quarkus" wizard & hit "Next"
2. ASSERT: project settings page is shown prompting to provide inputs in a form

**Result:**
grafik
The form has no border on the left

### LINKED ISSUE #637: "Select extension" wizard page has layout flaws
**Steps:**
1. EXEC: launch IDEA, hit "New Project", select "Quarkus" wizard, provide project settings and get to the extensions page

**Result:**
grafik
extensions settings page ohas neither left nor right borders

2. ASSERT: the title for the column "Selected extension" is left aligned
grafik
3. EXEC: select extension

**Result:**
grafik
Now that an extension is selected, the column title "Selected extensions" is right aligned

### PR DESCRIPTION
**Pull request by fbricon**
Screenshot 2023-06-22 at 15 13 03

Screenshot 2023-06-22 at 14 49 16

Screenshot 2023-06-22 at 14 49 09

fixes #636
fixes #637

### CODE REVIEW

**Comment by angelozerr** [comment]
For the Filter extension field (in last wizard page) I will grow up the field size to take the full horizontal  size of the page.

It is not linked to this issue but I have noticed that click on Next button is slow. I think it is because it collect Quarkus extension, but I think we should manage a progress monitor or something like this which explains that Quarkus extensions are collected. At first I though it didn't work and click on Next button several times.

File: src/main/java/com/redhat/devtools/intellij/quarkus/module/QuarkusExtensionsStep.java [hunk 1]
```diff
@@ -124,18 +107,23 @@ public QuarkusExtensionsStep(WizardContext wizardContext) {
...
             label1.setAlignmentX(Component.LEFT_ALIGNMENT);
             panel.add(label1);
             SearchTextField filter = new SearchTextField() {
                 @Override
                 public Dimension getMaximumSize() {
-                    Dimension size = super.getPreferredSize();
-                    size.height = JBUI.scale(30);
-                    return size;
+                    Dimension maxSize = super.getMaximumSize();
+                    return new Dimension(maxSize.width, JBUI.scale(30));
```
Comment (fbricon) [inline src/main/java/com/redhat/devtools/intellij/quarkus/module/QuarkusExtensionsStep.java:126]: Made the filter field fill the horizontal space:

Screenshot 2023-06-26 at 10 21 45

**Comment by adietish** [comment]
@fbricon:
image

**Review by adietish — APPROVED** [review]
LGTM.
Made first wizard page aka code endpoint chooser step only have left margin. Other than that +1

**Comment by fbricon** [comment]
@adietish definitely too much space, I spent some time trying to fix it already, but IMHO this requires rewriting the QuarkusModuleBuilder so it doesn't extend JavaModuleBuilder, so we can control the layout of the SDK field. I chose to ignore that in this PR for now, it will need to be addressed separately.

**Comment by fbricon** [comment]
> @adietish definitely too much space, I spent some time trying to fix it already, but IMHO this requires rewriting the QuarkusModuleBuilder so it doesn't extend JavaModuleBuilder, so we can control the layout of the SDK field. I chose to ignore that in this PR for now, it will need to be addressed separately.

I was actually thinking more of the SDK left alignment, more than the space below

**Comment by fbricon** [comment]
@adietish thanks

**Pull request merged**